scripts/updateDocs.sh

```diff
@@ -0,0 +1,42 @@
+set -eu
```

Comment:
Missing shebang line here: `#!/usr/bin/env bash`
Also, if there are pipes included in the script, it might be wise to use `set -eu -o pipefail` Otherwise  the exitcode of the last command will be used even if one of the inbetween commands fails.